mappings/net/minecraft/village/TradeOffers.mapping

```diff
@@ -101,6 +117,16 @@ CLASS net/minecraft/unmapped/C_hqrlptsj net/minecraft/village/TradeOffers
 			ARG 6 maxUses
 			ARG 7 experience
 			ARG 8 multiplier
```

Comment:
Same

Reply:
I mapped this, I'm pretty sure it's for experience (also cannot check source rn)

Reply:
checked source, it's for the price. Updated the name everwhere in the file!